app/measurement/views.py

```diff
@@ -132,9 +136,11 @@ def get_queryset(self):
         return Metric.objects.all()
 
 
+REQUIRED_MEASUREMENT_PARAMS = ("metric", "starttime", "endtime")
```

Comment:
Should add "channel" here I think. Definitely for the AggregateViewSet, double check for the MeasurementViewSet?